.eslintignore

```diff
@@ -20,6 +20,8 @@ package-lock.json
 !/resources/todomvc/vanilla-examples/javascript-es6
 !/resources/todomvc/vanilla-examples/javascript-es6-webpack
 
+!resources/todomvc/todomvc-css
```

Comment:
Are these lines necessary for the code to be checked by eslint? I think they're useful if there's another line that adds an ignore that matches it, but I don't think there's any for this one.

(same comment for prettierignore below)